Jointify/Model/PoseMachineLearning/PoseNet.swift

```diff
@@ -59,11 +59,16 @@ class PoseNet {
         static let jointColor: UIColor = UIColor.systemPink
         // Degrees that will be subtraced from the measured angle
         static let neutralNullAngle: Float = 90.0
+        // Minimum confidence that each joint has to conform to
```

Comment:
Is it confidence per joint or confidence per frame?

Reply:
Confidence per drawn joint in each frame. So for example if you want to analyse your right knee, we only use images with a confidence value > 0.7 for rightHip, rightKnee & rightAnkle. If one of these is less than 0.7 we reject the image.

Reply:
Please adjust the comment then

- [ ] adjusted comment

Reply:
Please adjust comment

Reply:
Please adjust comment accordingly 👍